src/control/cmd/dmg/shell.go

```diff
@@ -195,7 +196,7 @@ func setupShell() *ishell.Shell {
 				c.Println(err)
 			}
 
-			killRankSvc(poolUUID, uint32(rank))
+			killRankSvc(conns, poolUUID, uint32(rank))
```

Comment:
this command, and poolCreate/Destroy when added to shell, will currently be broadcast rather than being sent to an access point. we need to adjust the conns list before killRank gets called. This is something unrelated to the changes in this patch, just recording the fact.